parser/feed/src/main/java/de/danoeh/antennapod/parser/feed/PodcastIndexTranscriptParser.java

```diff
@@ -0,0 +1,191 @@
+package de.danoeh.antennapod.parser.feed;
+
+import android.util.Log;
+
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.jsoup.internal.StringUtil;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
+import de.danoeh.antennapod.model.feed.Transcript;
+import de.danoeh.antennapod.model.feed.TranscriptSegment;
+
+public class PodcastIndexTranscriptParser {
+
+    public static class PodcastIndexTranscriptSrtParser {
+
+        private static String TAG = "PodcastIndexTranscriptSrtParser";
+
+        public static Transcript parse(String str) {
+            Transcript transcript = new Transcript();
+
+            TranscriptSegment transcriptSegment;
+            List<String> lines = Arrays.asList(str.split("\n"));
```

Comment:
Could there maybe also be Windows line endings that would cause problems here?

Reply:
add `str = str.replaceAll("\r\n", "\n")`